src/Sentry/Internal/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,10 @@
+using System.Text.RegularExpressions;
+
+namespace Sentry.Internal.Extensions
+{
+    internal static class StringExtensions
+    {
+        public static string ToSnakeCase(this string str) =>
```

Comment:
The purpose and use cases of snake casing could maybe be documented as a comment as well 🐍